README.md

```diff
@@ -840,9 +842,40 @@ file authors to find the optimal name split in any given case.
 
 - Cf. [Entity objects](#entity-objects) for details.
 
+## Identifier objects
+
+An identifier object represents a persistent identifier. In CFF, identifier objects are realized as
+collections with two defined keys, both mandatory.
+
+| Identifier key      | Identifier data type     | optional       |
+| -               | -                    | :------------: |
+| `type`  | String ([*Identifier type string*](#identifier-type-strings))              |                |
+| `value`   | String               |                |
+
+Table: Complete list of keys for identifier objects.
```

Comment:
We can probably remove this, given that we have https://github.com/citation-file-format/citation-file-format/pull/96/files#diff-04c6e90faac2675aa89e2176d2eec7d8R955-R956 already.

Reply:
Wait I just realized it is the table caption.